.pre-commit-config.yaml

```diff
@@ -1,30 +1,33 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 23.1.0
-    hooks:
-      - id: black
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: "v0.0.254"
-    hooks:
-      - id: ruff
-  - repo: local
-    hooks:
-      - id: clean-notebooks
-        name: clean-notebooks
-        files: \.ipynb$
-        stages: [commit]
-        language: system
-        entry: jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "6f3cb139ef36133b6f903b97facc57b07cef57c9"
-    hooks:
-      - id: prettier
-  - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: "8ddcbd412c0b348841f0f82c837702f432539652"
-    hooks:
-      - id: eslint
-        additional_dependencies:
-          - eslint@8.31.0
-          - eslint-plugin-react@7.31.11
-          - eslint-plugin-react-hooks@4.6.0
-          - eslint-plugin-simple-import-sort@10.0.0
+    - repo: https://github.com/psf/black
+      rev: 23.1.0
+      hooks:
+          - id: black
+    - repo: https://github.com/charliermarsh/ruff-pre-commit
+      rev: "v0.0.254"
+      hooks:
+          - id: ruff
+    - repo: local
+      hooks:
+          - id: clean-notebooks
+            name: clean-notebooks
+            files: \.ipynb$
+            stages: [commit]
+            language: system
+            entry: jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace
+    - repo: https://github.com/pre-commit/mirrors-prettier
+      rev: "6f3cb139ef36133b6f903b97facc57b07cef57c9"
+      hooks:
+          - id: prettier
+            files: '\.(jsx?|tsx?|css)$'
+            additional_dependencies:
+                - prettier@2.8.7
```

Comment:
pre-commit is pretty bad at versions and only really supports pretter3 which is in alpha